fast-list.js

```diff
@@ -592,9 +677,12 @@ function computeIndices(source, geometry) {
 }
 
 function recycle(items, start, end, action) {
-  debug('recycle', start, end, action);
+  debug('recycle', items, start, end, action);
   var recyclableItems = [];
 
+  end--;
+  start--;
```

Comment:
RE Help: Mega hack, need help here! I don't quite understand the logic. I sometimes found that the `start` and `end` were `1` too large.